src/cli_parser.cpp

```diff
@@ -19,20 +19,17 @@
 #include <stdexcept>
 #include <string>
 
-// TODO: Write windows/linux specific status codes.
-#ifdef __linux__
-#include <sysexits.h>
-#elif _WIN32
-#include <ntstatus.h>
-#endif
 #include "capi_frontend/server_settings.hpp"
+#include "graph_export/graph_cli_parser.hpp"
+#include "ovms_exit_codes.hpp"
 #include "version.hpp"
 
 namespace ovms {
 
 void CLIParser::parse(int argc, char** argv) {
     try {
         options = std::make_unique<cxxopts::Options>(argv[0], "OpenVINO Model Server");
+        options->allow_unrecognised_options();
```

Comment:
Why we  need this? This will probably block simple validation like mismatchint `--model_name` with `--models_name`?

Reply:
Ah I see - we catch that in graph cli parser. Maybe 1 line comment why we do that?